.github/workflows/version_up_major.yaml

```
@@ -0,0 +1,43 @@
+name: Version Up Major
+
+on:
+  workflow_dispatch:
+
+jobs:
+  version-up:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)